cpp/src/barretenberg/stdlib/primitives/field/field.test.cpp

```diff
@@ -975,6 +975,52 @@ template <typename Composer> class stdlib_field : public testing::Test {
         bool proof_result = verifier.verify_proof(proof);
         EXPECT_EQ(proof_result, true);
     }
+
+    static void test_ranged_less_than()
+    {
+        Composer composer = Composer();
+
+        for (size_t i = 0; i < 10; ++i) {
+            uint8_t a_val = engine.get_random_uint8();
+            uint8_t b_val = 0;
+            switch (i) {
+            case 0: {
+                b_val = a_val;
+                break;
+            }
+            case 1: {
+                a_val = static_cast<uint8_t>(
```

Comment:
nit: the static_casts and ternary operators being done here hurt's readability.

Perhaps, we can have a zero_u8 and one_u8 variable outside of the for_loop and have the ternary statement be on its own line